src/utils/args.rs

```diff
@@ -94,6 +96,18 @@ impl<'a: 'b, 'b> ArgExt for Command {
         )
     }
 
+    fn org_arg_with_id(self) -> Command {
+        self.arg(
+            Arg::new("org")
+                .value_name("ORG")
+                .long("org")
+                .short('o')
+                .value_parser(validate_org)
```

Comment:
You are still using the same value parser as the `org` argument here. I think the validation itself should be correct, but if you pass an invalid value, it prints `"Invalid value for organization. Use the URL slug and not the name!"`, which is misleading.

Reply:
With the changes I requested, we would no longer have a separate `org_arg_with_id` function.

Instead, we would need to update the existing `validate_org` value parser to state that an organization slug _or ID_ needs to be provided.

Reply:
Of course, that also (like the rest of the changes) can only be done once all Sentry APIs support org or project IDs